cli/flags.rs

```diff
@@ -556,155 +517,28 @@ The declaration file could be saved and used for typing information.",
 
 fn fmt_subcommand<'a, 'b>() -> App<'a, 'b> {
   SubCommand::with_name("fmt")
-        .about("Format files")
-        .long_about(
-"Auto-format JavaScript/TypeScript source code using Prettier
+    .about("Format files")
+    .long_about(
+      "Auto-format JavaScript/TypeScript source code using dprint
```

Comment:
s/using dprint//
I don't think it's useful in the first sentence and probably confusing to people.